projects/core/src/user/occ/occ-user-address.adapter.ts

```diff
@@ -0,0 +1,95 @@
+import { Injectable } from '@angular/core';
+import { UserAddressAdapter } from '../connectors/address/user-address.adapter';
+import { Address, AddressValidation } from '../../model/address.model';
+import { Observable, throwError } from 'rxjs';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { catchError, map } from 'rxjs/operators';
+import { Occ } from '../../occ/occ-models/occ.models';
+import { OccEndpointsService } from '../../occ/services/occ-endpoints.service';
+import { ConverterService } from '../../util/converter.service';
+import {
+  ADDRESS_NORMALIZER,
+  ADDRESS_SERIALIZER,
+  ADDRESS_VALIDATION_NORMALIZER,
+} from '../connectors/address/converters';
+
+const USER_ENDPOINT = 'users/';
+const ADDRESSES_ENDPOINT = '/addresses';
+const ADDRESSES_VERIFICATION_ENDPOINT = '/addresses/verification';
+
+@Injectable()
+export class OccUserAddressAdapter implements UserAddressAdapter {
+  constructor(
+    protected http: HttpClient,
+    protected occEndpoints: OccEndpointsService,
+    protected converter: ConverterService
+  ) {}
+
+  private getUserEndpoint(userId: string): string {
+    const endpoint = `${USER_ENDPOINT}${userId}`;
+    return this.occEndpoints.getEndpoint(endpoint);
+  }
+
+  load(userId: string): Observable<Address[]> {
```

Comment:
Souldn't it be `loadList` to be consistent with payment methods adapter? Or the other way around?